.github/workflows/python-package.yaml

```diff
@@ -0,0 +1,42 @@
+name: Python package
+
+on:
+  push:
+    branches:
+  pull_request:
+    branches:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.10"]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          git rev-parse --short HEAD > hotsos/.repo-info
+          python -m pip install --upgrade pip
+          python -m pip install build
+          python -m build
+
+      - name: Publish a Python distribution to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        if: github.event_name == 'push' && github.ref_name == 'main' && matrix.python-version == '3.8'
```

Comment:
@dosaboy I added a few conditionals here so that this step should only be used at the right moment. A PR is not going to trigger this and I can't test it.

Whenever you have some time, please add the PyPi API token as `PYPI_API_TOKEN` in the repository secrets. That should allow this step here to work.